handbook/engineering/web/index.md

```diff
@@ -1,8 +1,8 @@
 # Web team
 
-The web team owns all work related to our web application and browser extensions that isn't already owned by one of the other mission based teams.
+The web team owns the maintenance and expansion of our web application and code host integrations as vehicles to deliver the value of [extensions](https://docs.sourcegraph.com/extensions) to our users.
 
-This is a large ownership area, so the team creates a focused plan each iteration, by agreeing on an appropriately small set of [goals](../../../company/goals/index.md). Each goal should have more than one teammate working on it.
+This is a large ownership area, so the team creates a focused plan each iteration, by agreeing on an appropriately small set of [iteration goals](../../../company/goals/index.md). Each goal should have more than one teammate working on it.
 
 The web team's current focus is documented in [the tracking issue for the current milestone](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+label%3Atracking+label%3Ateam%2Fweb+is%3Aopen).
```

Comment:
The goals section now feels like it is missing short term, but that is documented here. Should we move this sentence (and perhaps the previous) down to the goal section under a `### Short term` heading?